setup.sh

```diff
@@ -13,6 +13,10 @@ systemctl disable nginx
 
 python3 -m pip install -r requirements.txt
 
+# fixes problem if `ModuleNotFoundError: No module named 'cryptography.hazmat.backends.openssl.ec` (`scapy` related)
+python3 -m pip uninstall -y cryptography
+python3 -mpip install cryptography==38.0.2
```

Comment:
can't we add a library version to the requirements? why should we remove and reinstall it again?

Reply:
Updated.